packages/workflow/src/NodeErrors.ts

```diff
@@ -290,28 +290,21 @@ interface NodeApiErrorOptions extends NodeOperationErrorOptions {
  * with an HTTP error code, an error message and a description.
  */
 export class NodeApiError extends NodeError {
-	httpCode: string | null;
+	httpCode: string | null = null;
 
 	constructor(
 		node: INode,
 		error: JsonObject,
 		{ message, description, httpCode, parseXml, runIndex, itemIndex }: NodeApiErrorOptions = {},
 	) {
+		if (error.reason instanceof NodeApiError) return error.reason;
```

Comment:
this should be a separate PR. The issue this is solving is of us accidentally wrapping NodeApiError instances with new NodeApiError. This is what causes a lot of the "Unknown error" in the frontend.